challenges/web-frontend/libs/shared/apis/src/lib/auction/auction.service.ts

```diff
@@ -0,0 +1,46 @@
+import { HttpClient, HttpHeaders, HttpParams } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { IAuctionFilter } from '../models/auction.interface';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class AuctionService {
+  private baseUrl = 'https://api-core-dev.caronsale.de/api/v1';
+  private url = 'https://api-core-dev.caronsale.de/api/v1/auction/buyer';
+  private urlv2 = 'https://api-core-dev.caronsale.de/api/v2/auction/buyer';
```

Comment:
This URL needs a trailing slash. Without it, it gives only "401 Unauthorized".
```suggestion
  private urlv2 = 'https://api-core-dev.caronsale.de/api/v2/auction/buyer/';
```